internal/command/jsonprovider/schema.go

```diff
@@ -36,3 +36,35 @@ func marshalSchemas(schemas map[string]providers.Schema) map[string]*Schema {
 	}
 	return ret
 }
+
+type IdentitySchema struct {
+	Version    uint64                        `json:"version"`
+	Attributes map[string]*IdentityAttribute `json:"attributes,omitempty"`
```

Comment:
Should this be called `identity_attributes` or `attributes`?

Reply:
Given that it is under a struct called Ì`IdentitySchema`, `attributes` seem sufficient to me